src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -236,7 +236,7 @@ private DependencyType GetDependencyType(string dependencyTypeString)
 
                 var currentPackageUniqueId = $"{parentTargetId}/{currentItemId}";
                 // add current package to dependencies world
-                var currentItem = items[currentItemId];
```

Comment:
>items[currentItemId] [](start = 34, length = 20)

So the objects in `items` are never added to DependenciesWorld? I guess this is unnecessary extra work in Single-targeting projects(?) -- a (perhaps minor) performance enhancement may be to do this only on cross-targetting projects.